src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/ManageUserService.kt

```diff
@@ -2,11 +2,12 @@ package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service
 
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.client.manageusers.ManageUsersApiClient
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.client.manageusers.UserDetailsDto
 
 @Component
 class ManageUserService(
   private val manageUsersApiClient: ManageUsersApiClient,
 ) {
-  fun getUserDetails(username: String) =
+  fun getUserDetails(username: String): UserDetailsDto =
```

Comment:
I know with kotlin we dont need to do this type of thing and it will infer the types .... but sometimes I like to see it on the method sig so I know what it's returning (rather than clicking into the implementation etc to find out, which is what I had to do in this case)